inst/examples/showcase/app.R

```diff
@@ -163,7 +199,7 @@ addResourcePath("showcase-static", "./static")
 shinyApp(
   ui = tagList(
     tags$script(
-      src = "showcase-static/js/highlight.min.js"
+      src = "showcase-static/js/highlight.v11.7.0.min.js"
```

Comment:
[question, optional]
have you considered using a CDN?

This doesn't add that much size to the js so it's completely fine like this

Reply:
Previously Filip also downloaded source code of `prism.js` so I just automatically did the same.